api/migrations/dev-data/data.sql

```diff
@@ -3,645 +3,1163 @@
+	('bae69a1c9fcc', 'activated', NULL, '$2y$13$KTCSklVQHNvbwJQ3Awl8Ee7t0wJB1gfRBXDANeQlBblqwJ4wgOEmC', '2022-01-23 16:19:10', '2023-08-08 09:44:16', 'f9f1a2f9af25', NULL),
+    ('dac7116e02ed', 'activated', null, '$2y$13$KTCSklVQHNvbwJQ3Awl8Ee7t0wJB1gfRBXDANeQlBblqwJ4wgOEmC', '2023-08-12 16:43:34', '2023-08-12 16:43:34', '4bc441dc4b29', null),
+    ('566aea2c2759', 'activated', null, '$2y$13$KTCSklVQHNvbwJQ3Awl8Ee7t0wJB1gfRBXDANeQlBblqwJ4wgOEmC', '2023-08-12 16:59:38', '2023-08-12 16:59:38', '51245d0e2ad4', null),
+    ('a3d9d86dc23b', 'activated', null, '$2y$13$KTCSklVQHNvbwJQ3Awl8Ee7t0wJB1gfRBXDANeQlBblqwJ4wgOEmC', '2023-08-12 16:46:43', '2023-08-12 16:46:43', '5552108bf43e', null),
+    ('a2f4f3879c85', 'activated', null, '$2y$13$KTCSklVQHNvbwJQ3Awl8Ee7t0wJB1gfRBXDANeQlBblqwJ4wgOEmC', '2023-08-12 16:55:28', '2023-08-12 16:55:28', '3f3fa9319dd2', null),
 
 
 
 INSERT INTO public.camp (id, campprototypeid, isprototype, name, title, motto, addressname, addressstreet, addresszipcode, addresscity, createtime, updatetime, creatorid, ownerid, couponkey, organizer, kind, coachname, coursenumber, coursekind, trainingadvisorname, printyslogoonpicasso) VALUES
 	('6430aecc5422', NULL, false, 'CampUnrelated', 'vero', 'Officia id corporis incidunt saepe provident esse hic eligendi.', 'quos', '57654 Ondricka Trace Suite 792
 Port Asiaton, MN 44798-0182', '10529', 'New Kaylinfort', '2022-01-23 16:19:10', '2022-01-23 16:19:10', '3b41dca5c568', '3b41dca5c568', NULL, NULL, NULL, NULL, NULL, NULL, NULL, false),
-	('05ce4b9836e9', NULL, false, 'Sola 2023', 'Harry Potter Lager', 'Harry Potter', 'Oskar Hugentobler', 'Rainweg 23', '1999', 'Les Bois', '2022-01-23 16:19:10', '2022-02-04 19:01:39', 'bae69a1c9fcc', '9145944210a7', NULL, NULL, NULL, NULL, NULL, NULL, NULL, false),
 	('e5027d852487', NULL, true, 'J+S', 'J+S', '', '', '', '', '', '2022-01-23 16:19:10', '2022-02-04 21:45:19', '3b41dca5c568', '3b41dca5c568', NULL, NULL, NULL, NULL, NULL, NULL, NULL, false),
-	('c4cca3a51342', 'e5027d852487', false, 'Pfila 2023', 'GRGR', 'Grandiose RealityTV Show', NULL, NULL, NULL, NULL, '2022-02-04 21:59:10', '2022-02-04 21:59:10', '9145944210a7', '9145944210a7', NULL, NULL, NULL, NULL, NULL, NULL, NULL, false);
+	('3c79b99ab424', 'e5027d852487', false, 'Pfila 2023', 'GRGR', 'Grandiose RealityTV Show', 'Adrian Sonderegger', 'Bümplizstrasse 5', '3156', 'Bümpliz', '2023-08-08 07:53:12', '2023-08-08 07:56:11', '9145944210a7', '9145944210a7', NULL, 'Abteilung Gryfenberg', 'Pfingstlager', 'Andrea Schneider', NULL, NULL, NULL, false),
+	('6973c230d6b1', 'e5027d852487', false, 'Sola 2023', 'Harry Potter Lager', 'Harry Potter', 'Oskar Hugentobler', 'Rainweg 23', '1999', 'Les Bois', '2023-08-08 09:22:58', '2023-08-08 09:23:39', '9145944210a7', '9145944210a7', NULL, 'Pfadi Gryfenberg', 'Sommerlager', 'Andrea Schneider', NULL, NULL, NULL, false),
```

Comment:
Is this public data?

Reply:
This is invented by me.
I hope i did not accidentally hit real world data.